test/unittests/commands/xmllib/test_value_converters.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+
+import pandas as pd
+import pytest
+
+from dsp_tools.commands.xmllib.value_converters import convert_to_bool_string
+
+
+@pytest.mark.parametrize("val", ["false", "0  ", "  0.0", "NO", False])
+def test_convert_to_bool_false(val: Any) -> None:
+    assert convert_to_bool_string(val) == "false"
+
+
+@pytest.mark.parametrize("val", ["TRUE ", "  1", "1.0", "Yes", True])
+def test_convert_to_bool_true(val: Any) -> None:
+    assert convert_to_bool_string(val) == "true"
+
+
+@pytest.mark.parametrize("val", [pd.NA, None, 2.421, 10, "other", "  "])
+def test_convert_to_bool_failure(val: Any) -> None:
+    assert convert_to_bool_string(val) == str(val)
```

Comment:
Converting this is a wish from RDU. I separated conversion and checking strictly, so there is a warning raised in the `__post__init__`. So if the input is wrong, I want them to keep the input. Raising errors is unpopular, so I wont' do that.

Reply:
So, RDU wishes that the scripts produce an XML file with `<boolean>2.421</boolean>`, and only a warning is emitted, instead of an error? I'm a bit surprised, but if this is intended, then okay.

Reply:
Yes, it will not pass XML validation, so it does no harm.
I think it is because they are able to more quickly fix the errors if they have all of them collected and they see it concretely and not only on the console...